glassfish-runner/signature/platform_pom.xml

```diff
@@ -179,15 +187,15 @@
                                 <artifactItem>
                                     <groupId>jakarta.tck</groupId>
                                     <artifactId>signaturevalidation</artifactId>
-                                    <version>${version.jakarta.tck}</version>
+                                    <version>${tck.version}</version>
                                     <overWrite>true</overWrite>
                                     <outputDirectory>${base.tck.dir}</outputDirectory>
                                     <includes>**/sig-test.map,**/sig-test-pkg-list.txt</includes>
                                 </artifactItem>
                                 <artifactItem>
                                     <groupId>jakarta.tck</groupId>
                                     <artifactId>signaturevalidation</artifactId>
-                                    <version>${version.jakarta.tck}</version>
```

Comment:
Good catch!

Reply:
Yes I just made those changes. Thanks for reminding! 
But to get the tests running with milestone versions we would need to remove the maven resolver in the signature test sources.